simulation/src/test/java/com/palantir/dialogue/core/SimulationTest.java

```diff
@@ -430,19 +441,23 @@ public void after() throws IOException {
                     "%s success=%.0f%% client_mean=%.1f ms server_cpu=%s",
                     strategy, result.successPercentage(), clientMeanMillis, serverCpu));
             XYChart serverRequestCount = simulation.metrics().chart(Pattern.compile("request.*count"));
-            XYChart clientStuff = simulation.metrics().chart(Pattern.compile("(refusals|starts).count"));
+            // XYChart clientStuff = simulation.metrics().chart(Pattern.compile("(refusals|starts).count"));
```

Comment:
remove?

Reply:
I actually wanna keep this because toggling it is useful for local debugging.